blog/locale/uk/LC_MESSAGES/django.po

```diff
@@ -8,7 +8,7 @@ msgid ""
 msgstr ""
 "Project-Id-Version: PACKAGE VERSION\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2018-02-10 15:49+0200\n"
+"POT-Creation-Date: 2018-02-20 11:29+0200\n"
```

Comment:
You might want to add messages generation to CI and then check whether there's new strings, for which you forgot to run and commit re-regerated po file.